masterfabric-mobile/swift-camp
# CL - Added SwiftUI Text Component Implementation

## 📋 PR Description

Implemented SwiftUI Text component with basic functionality and integrated into the existing structure.

---

## ✅ Checklist


---

## 🛠 Steps to Test

1. Navigate to the screen with the Text component and ensure it displays correctly with the expected content and style.
2. Verify that the implementation works without errors or warnings in different devices and orientations.
---

## 🔗 Related Links

- Documentation: https://medium.com/@dickareynaldisiahaan/viper-architecture-in-swift-building-robust-ios-apps-ed30d673d459

---

### 📷 Screenshots (Optional)

N/A
